app/models/geo_local_authority.rb

```diff
@@ -0,0 +1,31 @@
+class GeoLocalAuthority < ApplicationRecord
+  include RGeo::ActiveRecord::GeometryMixin
+
+  SRID = 27_700
+
+  def self.nearest_three_to(string)
```

Comment:
Maybe `self.nearest_to(string, amount: 3)` instead hardcoding argument in method name?